stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/ChatDomainImpl.kt

```diff
@@ -259,7 +260,16 @@ internal class ChatDomainImpl internal constructor(
         initClean()
     }
 
+    internal val job = SupervisorJob()
+    internal var scope = CoroutineScope(job)
+    internal var scopeIO: CoroutineScope
+    internal var scopeMain: CoroutineScope
+
     init {
+        dispatcherIO = customDispatcherIO ?: Dispatchers.IO
+        dispatcherMain = customerDispatcherMain ?: Dispatchers.Main
+        scopeIO = CoroutineScope(job + dispatcherIO)
```

Comment:
So do we still need these two scopes?

Reply:
this was just a shortcut for running on main, it's not needed